### PR TITLE
Add flags to enable TF2 infer/batcher test

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: qa/common/sequence_util.py [hunk 1]
```diff
@@ -51,8 +51,7 @@
     int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
 _test_cuda_shared_memory = bool(
     int(os.environ.get('TEST_CUDA_SHARED_MEMORY', 0)))
-_test_valgrind = bool(
-    int(os.environ.get('TEST_VALGRIND', 0)))
+_test_valgrind = bool(int(os.environ.get('TEST_VALGRIND', 0)))
```
Comment (deadeyegoodwin) [inline qa/common/sequence_util.py:54]: Why this change?

**Comment by deadeyegoodwin** [comment]
What about L0_sequence_batcher?

File: qa/common/sequence_util.py (hunk 1 above)
Comment (CoderHam) [reply qa/common/sequence_util.py:54]: Picked up by formatter.

**Comment by CoderHam** [comment]
> What about L0_sequence_batcher?

The sequence batcher is still failing on Jetson. I can add support for enabling TF2 testing on x86 but I want to hold off enabling it on jetson till the timing related failures are resolved on Jetson.

File: qa/common/sequence_util.py (hunk 1 above)
Comment (deadeyegoodwin) [reply qa/common/sequence_util.py:54]: ok

**Comment by deadeyegoodwin** [comment]
I would add it as part of this change. You don't need to add a jetson test that uses it until it is working. For x86 I filed a separate ticket and for x86 we just want to run L0_infer.

**Comment by CoderHam** [comment]
> I would add it as part of this change. You don't need to add a jetson test that uses it until it is working. For x86 I filed a separate ticket and for x86 we just want to run L0_infer.

Done

**Review by deadeyegoodwin — APPROVED** [review]
